src/views/ChallengeList.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <v-row v-if="!loading">
+    <v-col>
+      <v-chip label variant="text" prepend-icon="mdi-tag-outline">
+        {{ $t('Challenge.ChallengeCount', { count: challengeTotal }) }}
+      </v-chip>
+      <LoadedCountChip :loadedCount="challengeList.length" :totalCount="challengeTotal" />
+      <FilterMenu kind="challenge" :currentFilter="currentFilter" :currentType="currentType" @update:currentFilter="toggleChallengeFilter($event)" @update:currentType="toggleChallengeType($event)" />
+    </v-col>
+  </v-row>
+
+  <v-row class="mt-0">
+    <v-col v-for="challenge in challengeList" :key="challenge" cols="12" sm="6" md="4" xl="3">
+      <v-card :id="'challenge_' + challenge.id" :class="'border-transparent'" @click="goToChallenge(challenge)">
+        <template #title>
+          {{ challenge.icon }} {{ challenge.title }} {{ challenge.subtitle }}
+        </template>
+
+        <v-divider />
+
+        <v-card-text>
+          <p class="mb-2">
+            {{ $t('Challenge.ChallengeStatus', { status: $t('Challenge.ChallengeStatuses.' + challenge.status) }) }}
+          </p>
+          <DateChip :date="challenge.start_date" />
+          <DateChip :date="challenge.end_date" />
+          <p v-if="challenge.categories.length">
+            <CategoryTagChip v-for="category in challenge.categories" :key="category" :category="{id: category, name: category}" />
+          </p>
+        </v-card-text>
+      </v-card>
+    </v-col>
+  </v-row>
+
+  <v-row v-if="loading">
+    <v-col align="center">
+      <v-progress-circular indeterminate :size="30" />
+    </v-col>
+  </v-row>
+</template>
+
+<script>
+import { defineAsyncComponent } from 'vue'
+import api from '../services/api'
+import constants from '../constants'
+import utils from '../utils.js'
+
+export default {
+  components: {
+    LoadedCountChip: defineAsyncComponent(() => import('../components/LoadedCountChip.vue')),
+    FilterMenu: defineAsyncComponent(() => import('../components/FilterMenu.vue')),
+    CategoryTagChip: defineAsyncComponent(() => import('../components/CategoryTagChip.vue')),
+    DateChip: defineAsyncComponent(() => import('../components/DateChip.vue'))
+  },
+  data() {
+    return {
+      challengeList: [],
+      challengeTotal: null,
+      challengePage: 0,
+      loading: false,
+      currentFilter: '',
+      currentType: '',
+      currentOrder: 'id',
+    }
+  },
+  computed: {
+    getChallengesParams() {
+      let defaultParams = { order_by: this.currentOrder, page: this.challengePage, status: "ONGOING" }
+      if (this.currentFilter === 'show_all_status') {
```

Comment:
not sure if having this filter (and especially filtered by default) is a good idea. rather load them all but display them in sections ? I was surprised on first load to have 0 challenges being displayed